Signal/src/util/NSAttributedString+OWS.h

```diff
@@ -4,6 +4,12 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+@interface NSString (OWS)
+
+- (NSString *)rtlSafeAppend:(NSString *)string referenceView:(UIView *)referenceView;
+
+@end
```

Comment:
I skipped creating a separate source file for this category.

Reply:
nit: can you move this to the existing `NSString+OWS.h`?

Reply:
Oh good point!